### PR TITLE
Initialize firebase cloud functions

### PR DESCRIPTION
**Pull request by qdozaq**
This is just the base initialization from running `firebase init functions`

Lookin to use these for paypal integrations.

### CODE REVIEW

File: functions/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,2388 @@
+{
```
Comment (azmainamin) [inline functions/package-lock.json:1]: Should the lock file be this big?

File: functions/.gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Compiled JavaScript files
+**/*.js
```
Comment (azmainamin) [inline functions/.gitignore:2]: Just to verify, should we ignore .js files as well?

Comment (qdozaq) [reply functions/.gitignore:2]: This glob should handle everything including *.js

File: functions/package-lock.json (hunk 1 above)
Comment (qdozaq) [reply functions/package-lock.json:1]: Yeh this is just from the default firebase and typescript modules. I guess they just have hella dependencies

**Pull request merged**